openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -228,22 +239,25 @@ def get_mce_asce7(prob_mce, det_imt, DLLs, dstore):
             crs = rtgm['RiskCoeff'][i]
         elif imt == b'SA1P0':
             cr1 = rtgm['RiskCoeff'][i]
-            
+
     det_mce = {}
     mce = {}  # imt -> MCE
     prob_mce_out = {}
     for i, imt in enumerate(det_imt):
         det_mce[imt] = max(det_imt[imt], DLLs[i])
-        mce[imt] = min(prob_mce[i], det_mce[imt]) 
+        mce[imt] = min(prob_mce[i], det_mce[imt])
+        # FIXME
```

Comment:
Why this comment? Is it relevant or it can be removed?

Reply:
I removed it